zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -26,11 +27,12 @@ zipkin:
       password: ${ACTIVEMQ_PASSWORD:}
     http:
       # Set to false to disable creation of spans via HTTP collector API
-      enabled: ${HTTP_COLLECTOR_ENABLED:true}
+      enabled: ${COLLECTOR_HTTP_ENABLED:true}
```

Comment:
This is part of a proposal to have enabled variables under `COLLECTOR_XXX_ENABLED`

Reply:
sadly we'll need to nest the old variable. I forget the exact syntax, but it can be done. Then update the READMEs

Reply:
Not sure which we want to give precedent to if both exist, but the `:` is the "else" operator here, so giving precedent to the old one should be like `${HTTP_COLLECTOR_ENABLED:${COLLECTOR_HTTP_ENABLED:true}}`

Reply:
Thanks! Adding that now.